tests/functional/processes/file/test_fulfillment_process.py

```diff
@@ -0,0 +1,111 @@
+from processes import ClusterProcess, FulfillURLManifestProcess
+from model import Record, Item, Edition, Work, Link
+import json
+import os
+from sqlalchemy.orm import joinedload
+
+
+def test_fulfillment_process(db_manager, s3_manager, seed_test_data):
+
+    manifest_record_uuid = seed_test_data['manifest_record_uuid']
+
+    cluster_process = ClusterProcess('complete', None, None, manifest_record_uuid, None)
+
+    unclustered_manifest_record = db_manager.session.query(Record).filter(Record.uuid == manifest_record_uuid).first()
+    assert unclustered_manifest_record.cluster_status == False
+
+    cluster_process.runProcess()
+
+    db_manager.session.commit()
+
+    clustered_manifest_record = db_manager.session.query(Record).filter(Record.uuid == manifest_record_uuid).first()
+
+    assert clustered_manifest_record.cluster_status == True
+
+    frbrized_model = (
+        db_manager.session.query(Item, Edition, Work)
+            .join(Edition, Edition.id == Item.edition_id)
+            .join(Work, Work.id == Edition.work_id)
+            .filter(Item.record_id == clustered_manifest_record.id)
+            .first()
+    )
```

Comment:
You don't have to get the entire frbrized model. We should just be checking the links!

You can get the item associated with the record by querying the items table given record_uuid